h2o-core/src/main/java/water/Scope.java

```diff
@@ -92,4 +89,7 @@ static public void untrack(Iterable<Key<Vec>> keys) {
     HashSet<Key> xkeys = scope._keys.peek();
     for (Key<Vec> key : keys) xkeys.remove(key); // Untrack key
   }
+
+  @Override
```

Comment:
It has the following meaning. Scope now implements Closeable - on another request. Due to this, it needs a method called close(). Why it is important? In certain versions of Java, this provides automatic disposal of resources, by automatic call of close(). 
While I understand the principle called YAGNI, i'd rather look into the future, and not automatically require to delete anything I don't see fit my today's purposes. Or else we'll be like these guys in the autoshop: http://thedailywtf.com/articles/no-account-for-you